lambda-notes/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+
+import NavBar from "./Components/NavBar";
+
+import './App.css';
+import { notEqual } from 'assert';
```

Comment:
What are you planning to use this for?

Reply:
it looks like something auto-populated and automatically imported that. It's been removed in my latest push!